app/validators/url_validator.rb

```diff
@@ -0,0 +1,13 @@
+class UrlValidator < ActiveModel::Validator
+  def validate(record, attribute, value)
+    value = false
+    begin
+      valid = URI(value).is_a?(URI::HTTP)
+    rescue URI::InvalidURIError
+      valid = false
+    end
+    unless valid
+      record.errors[attribute] << (options[:message] || "is not a URL")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.